suite/MC/RISCV/insn-riscv32.s.cs

```diff
@@ -1,3 +1,4 @@
 # CS_ARCH_RISCV, CS_MODE_RISCV32, None
-0x37,0x34,0x00,0x00=lui s0, 3
-0x97,0x82,0x00,0x00=auipc t0, 8
+0x37,0x34,0x00,0x00 = lui	s0, 3
```

Comment:
oh no, just 1 space here.

Reply:
I am testing, figure out the problem, looks like the TAB does not come from *.s.cs files.

Reply:
Help, still can not figure out where is the TAB comes from.